llarp/util/logging/logger.hpp

```diff
@@ -83,14 +83,22 @@ namespace llarp
   struct LogContext
   {
     LogContext();
-    LogLevel minLevel = eLogInfo;
+    LogLevel curLevel     = eLogInfo;
+    LogLevel startupLevel = eLogInfo;
+    LogLevel runtimeLevel = eLogWarn;
     ILogStream_ptr logStream;
     std::string nodeName = "lokinet";
 
     const llarp_time_t started;
 
     static LogContext&
     Instance();
+
+    void
+    DropToRuntimeLevel();
+
+    void
+    RevertRuntimeLevel();
```

Comment:
The convention "startup level" and "runtime level" is very clear to me, this function name seems ambiguous though